### ISSUE TITLE
XML fails syntax check

### ISSUE THREAD

**Issue by DavidHaslam** [lemtom/coverdale#1]
I was about to test to see if the OSIS was valid to the official **2.1.1** schema¹, but before that it fails basic XML syntax check.

Then I noticed that there are **9119** instances of `&` in the text.
XML files should use the entity `&amp;` in its place.

After replacing these, the file still fails XML syntax check.
_Further investigation required._

#### Note:
1. The BibleTechnologies site has gone AWOL, so the schema location is currently inaccessible.
Meanwhile, you can use [one](http://www.crosswire.org/~dmsmith/osis/osisCore.2.1.1-cw-latest.xsd) on the CrossWire server.
Change line 2 to
```
<osis xmlns="http://www.bibletechnologies.net/2003/OSIS/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.bibletechnologies.net/2003/OSIS/namespace http://www.crosswire.org/~dmsmith/osis/osisCore.2.1.1-cw-latest.xsd">
```

**Comment by DavidHaslam** [comment]
Got it!

There were actually **9131** instances of `&`. The lower count was for a _whole words_ search.

Corrected 2 instances of missing space in `,&` before doing the replace all.

The XML file is now OK as regards syntax testing.

Just found that I cannot access the OSIS schema at CrossWire!
_Maybe the server is down for a maintenance task? I have asked DM._

Tried again with local copy of `osisCore.2.1.1.xsd`. 

The **osisWork** attribute is missing!  Changed line 5 to:
```
      <work osisWork="Coverdale">
```
Now validates!  Hurrah!

I will fork your repo., make these corrections and issue a pull request.

May take a while. Got a few other things on the go as well.

**Comment by DavidHaslam** [comment]
Fixed by merge request #8

**Comment by lemtom** [comment]
Thank you.
